docs/guides/custom-images.rst

```diff
@@ -0,0 +1,78 @@
+Using Custom Docker Images on Compute Engine
+============================================
+
+When running workflows remotely on Compute Engine, your tasks run in a container that uses one of the
+``orquestra-sdk-base`` images depending on whether you have requested a GPU or not (you can see their source code
```

Comment:
nit: instead of saying
> depending whether you have requested a GPU or not

I suggest changing to
> depending whether the `sdk.task` requires a GPU resource or not